lnrpc/rpc.proto

```diff
@@ -461,6 +461,10 @@ message SendRequest {
     payment to the recipient.
     */
     string payment_request = 6;
+
+    // Maximum fee to pay in millisatoshis. Routes with fees greater
+    // than max_fee will be abandoned during path finding.
+    int64 max_fee = 7 [json_name = "max_fee"];
```

Comment:
The initial issue specifies the ability to configure either: a max fee (absolute), or a ratio. Is there a precise reason you opted to only go with the first option?

Reply:
In my original PR I had a `oneof` field which allowed the user to specify all three of the options suggested in the issue. I am realizing I should not have squashed this issue down to one commit, so you could have more context.

I went with the singular field after a conversation with halseth where he argued "that this logic could be moved to the caller, such that we can have only the limit_satoshis field, and make the caller do the necessary calculation." You can see that conversation here:
https://github.com/lightningnetwork/lnd/pull/320#discussion_r139392829

I can certainly bring back the `oneof` if you think that would be best.